UI/MapControlSample/MapControlSample/MapControlSample.Skia.Gtk/MapControlSample.Skia.Gtk.csproj

```diff
@@ -21,8 +21,12 @@
 		<PackageReference Include="Uno.WinUI.RemoteControl" Version="4.9.26" Condition="'$(Configuration)'=='Debug'" />
 		<PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.9.26" />
 		<PackageReference Include="Uno.WinUI.Lottie" Version="4.9.26" />
-		<PackageReference Include="SkiaSharp.Views.Uno.WinUI" Version="2.88.3" />
 		<PackageReference Include="SkiaSharp.Skottie" Version="2.88.3" />
+
+		<PackageReference Include="SkiaSharp" Version="2.88.3" />
+		<PackageReference Include="SkiaSharp.HarfBuzz" Version="2.88.3" />
+		<PackageReference Include="SkiaSharp.NativeAssets.WebAssembly" Version="2.88.3" />
+		<PackageReference Include="SkiaSharp.Views.Uno.WinUI" Version="2.88.3" />
```

Comment:
Curious why these weren't needed with net6.0 but now needed?

Reply:
Required to fix the canary